### DISCUSSION TITLE
[Issue] 歌词翻译错位

### DISCUSSION THREAD

**Discussion by ProjectNyaser** [maotoumao/MusicFree#223] [Show and tell]
![Screenshot_2024-05-21-23-27-12-523_fun.upup.musicfree.jpg](https://github.com/maotoumao/MusicFree/assets/61904116/e0b59c15-bce0-43e4-8072-db2d426159f3)

如图所示，翻译向下移两句就正常了，_似乎有翻译的歌曲都会这样_。

**Comment by ProjectNyaser** [comment]
![Screenshot_2024-05-21-23-37-28-471_fun.upup.musicfree.jpg](https://github.com/maotoumao/MusicFree/assets/61904116/5c70d22f-303d-4b73-b489-33b51201a5f4)

这个要下移三句才正常（）

**Comment by maotoumao** [comment]
大概是因为歌词解析有点小问题… 提个issue吧~

**Reply by ProjectNyaser** [reply]
虽迟但到...提了捏~

**Comment by maotoumao** [comment]
已知问题🤣写了个bug 下个版本修复



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年06月29日 22:09 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [maotoumao/MusicFree] [Issue] 歌词翻译错位 (Discussion #223) |

Closed #223 as resolved.
